README.md

```diff
@@ -60,36 +60,37 @@ Now run Go launcher and navigate to "http://localhost:8000/quick-launch?url=" pa
 e.g."http://localhost:8000/quick-launch?url=http://localhost:7777/1_0001.json"
 ```
 
-### Deployment with [Helm](https://helm.sh/)
+### Deployment with [Concourse](https://concourse-ci.org/)
 
-To deploy this application with helm, you must have a kubernetes cluster already running and be logged into the cluster.
-
-Log in to the cluster using:
-```
-gcloud container clusters get-credentials survey-runner --region <region> --project <gcp_project_id>
-```
-
-You need to have Helm installed locally
-
-1. Install Helm with `brew install kubernetes-helm` and then run `helm init --client-only`
-
-1. Install Helm Tiller plugin for tillerless deploys `helm plugin install https://github.com/rimusz/helm-tiller`
+To deploy this application with Concourse, you must have a Kubernetes cluster and be logged in to a Concourse instance.
 
 ---
 
-The following environment variables can be set when deploying the app.
+The following environment variables should be set when deploying the app:
 - RUNNER_URL
-- DOCKER_REGISTRY *(optional)*
-- IMAGE_TAG *(optional)*
+- PROJECT_ID
+
+The following are optional variables that can also be set if needed:
+- DOCKER_REGISTRY
+- IMAGE_TAG
+- REGION
 
 To deploy to a cluster you can run the following command
 
+```sh
+RUNNER_URL=<runner_instance_url> \
+PROJECT_ID=<project_id> \
+fly -t <target_concourse_instance> execute \
+  --config ci/deploy.yaml --input eq-questionnaire-launcher-repo=.
```

Comment:
As per Andrews comment on runner, we should rename the input so the `--input` isn't needed.